app/connection/ConnectionManager.js

```diff
@@ -110,6 +115,13 @@ class ConnectionManager extends EventEmitter {
         }, timeout);
     }
 
+    _sendBufferedRequests() {
```

Comment:
can you add a `console.error` message here? I would like to know whether this method would ever get called, since it only appears in TrackJS. Be sure to indicate it's not an error.

Reply:
@brucebinary it happens in a very rare case. you have to go offline when chart is not loaded yet.
![image](https://user-images.githubusercontent.com/3080372/49500592-4c0c9800-f886-11e8-8a78-88bcc10e790a.png)